CONTRIBUTING.md

```diff
@@ -0,0 +1,68 @@
+# Ferrite.jl contributor guide
+
+## Documentation
+
+Contributing to documentation is a great way to get started with any new project. As a new
+user you have a unique perspective of what things need to be documented and explained better
+-- if something confuses you, chances are you're not alone. Remember that also simple
+changes like fixing typos are welcome contributions. If you are looking for specific things
+to work on you can look at [open issues][open-issues].
+
+Small changes can be done easily in GitHub's web interface (see [Editing
+files][gh-edit-files]). Every page in the documentation have an `Edit on GitHub` button at
+the top, which takes you to the correct source file.
+
+Making larger changes is easier locally after cloning the repository. With a local
+repository you can also preview the changes in your own browser. After starting a Julia REPL
+in the root of the repository you can execute the following snippet:
+
+```julia
+include("docs/liveserver.jl")
+```
+
+This uses [`LiveServer.jl`][liveserver] to launch a local webserver which you can visit at
+[http://localhost:8000](http://localhost:8000). `LiveServer.jl` will monitor changes to the
+source files. When you make a change, and save the file, `LiveServer.jl` will automatically
+rebuild the documentation, and automatically refresh your browser window. Note that the
+first build may take longer, but subsequent runs should be substantially faster and give you
+almost instant feedback in the browser.
+
+**Useful resources**
+ - General information about documenting Julia code in the [Julia manual][julia-doc].
+ - [Documentation for `Documenter.jl`][documenter] which is used to render the HTML pages.
+ - [Documentation for `Literate.jl`][literate] which is used for tutorials/examples.
+
+
+## Reporting issues
+
+If you have found a bug or a problem with Ferrite.jl you can open an [issue][new-issue]. Try
+to include as much information about the problem as possible and preferably some code that
+can be copy-pasted to reproduce it (see [How to create a Minimal, Reproducible
+Example][so-mre]).
+
+If you can identify a fix for the bug you can submit a pull request without first opening an
+issue, see [Code changes](#code-changes).
```

Comment:
Maybe we should add a short reminder that issues (and discussions) should be first searched before sibmitting a new issue?

Reply:
We don't have many problems with handling duplicates in this project yet, do we? It can be a bit difficult for beginners to see if something is a duplicate. I'd rather have a duplicate issue (which shouldn't be hard to identify for us) than someone not opening one when encountering a problem. If we add such a scentence I'd suggest to also add something along the lines of 
"If you are in doubt whether your issue is new, please do open an issue for it."

Reply:
I agree with Kim -- traffic is so low anyway. Experienced users will look for duplicates anyway, and for new users it might be a nice opportunity for us to "meet" them.